--- popt-1.16/Makefile.am~	2010-05-04 22:55:54.000000000 +0200
+++ popt-1.16/Makefile.am	2010-12-11 23:31:40.889824367 +0100
@@ -47,7 +47,7 @@ usrlib_LTLIBRARIES = libpopt.la
 libpopt_la_SOURCES = popt.c poptparse.c poptconfig.c popthelp.c poptint.c
 libpopt_la_LDFLAGS = -no-undefined @LTLIBINTL@ @LTLIBICONV@
 
-pkgconfigdir = $(prefix)/lib/pkgconfig
+pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = popt.pc
 
 if HAVE_LD_VERSION_SCRIPT
